dashboard/dashboard.py

```diff
@@ -2,6 +2,81 @@
 
 import re
 import streamlit as st
+import psycopg2
+from os import environ
+from dotenv import load_dotenv
+from psycopg2.extras import RealDictCursor
+
+
+def get_connection() -> psycopg2.connect:
+    """Returns connection to RDS."""
+    return psycopg2.connect(
+        dbname=environ.get("DB_NAME"),
+        host=environ.get("DB_HOST"),
+        user=environ.get("DB_USERNAME"),
+        password=environ.get("DB_PASSWORD"),
+        port=environ.get("DB_PORT")
+    )
+
+
+def get_cursor(conn) -> RealDictCursor:
+    """Returns cursor object to query RDS."""
+    return conn.cursor(cursor_factory=RealDictCursor)
+
+
+def get_genres_from_db() -> dict:
+    """
+    Returns a dictionary of all genres and their
+    corresponding IDs.
+    """
+    conn = get_connection()
+    cursor = get_cursor(conn)
+
+    cursor.execute("""SELECT genre_name, genre_id FROM genre
+                   ORDER BY genre_id;""")
+    genre_rows = cursor.fetchall()
+    print("GENRES\n")
+    genre_dict = {row['genre_name']: row['genre_id'] for row in genre_rows}
+
+    return genre_dict
+
```

Comment:
To expand on ben's point about print statements, might be useful to put some logging in so we can track what is happening in the file?